### PR TITLE
Updated documentation for using search pipeline for multiple indices

### PR DESCRIPTION
**Pull request by owaiskazi19**
### Description
Documentation for https://github.com/opensearch-project/OpenSearch/pull/13276

### Issues Resolved
Resolves https://github.com/opensearch-project/documentation-website/issues/7035

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by owaiskazi19** [comment]
@msfroh @dbwiddis can you take a look? Thanks

File: _search-plugins/search-pipelines/using-search-pipeline.md [hunk 1]
```diff
@@ -130,6 +130,53 @@ The response contains only the public document, indicating that the pipeline was
 ```
 </details>
 
+You can also search on multiple indices having the same default pipeline. For example, the below alias1 has 2 indices my_index1 and my_index2 and both the indices have the same default pipeline `my_pipeline` attached to them.
```
Comment (minalsha) [inline _search-plugins/search-pipelines/using-search-pipeline.md:133]: Change "For example, the below alias1 has 2 indices my_index1 and my_index2 and both the indices have the same default pipeline `my_pipeline` attached to them."

To "For example, below **alias1** has 2 indices **my_index1** and **my_index2** and both indices have same default pipeline `my_pipeline` attached to them."

Comment (dbwiddis) [reply _search-plugins/search-pipelines/using-search-pipeline.md:133]: +1

**Review by dbwiddis — APPROVED** [review]
LGTM with Minal's edit.

**Review by vagimeli — APPROVED** [review]
Doc review complete with edits for clarity and style. Moving to editorial review

File: _search-plugins/search-pipelines/using-search-pipeline.md [hunk 2]
```diff
@@ -130,6 +130,53 @@ The response contains only the public document, indicating that the pipeline was
 ```
 </details>
 
+You can search across multiple indexes that share the same default pipeline. For example, `alias1` has two indexes, `my_index1` and `my_index2`, both of which have the default pipeline `my_pipeline` attached to them.
```
Comment (natebower) [inline _search-plugins/search-pipelines/using-search-pipeline.md:133]: Should this end with a colon?

**Review by natebower — COMMENTED** [review]
@owaiskazi19 @vagimeli Just one comment. Thanks!

**Pull request merged**